src/common/signalk/SKNMEA2000Parser.cpp

```diff
@@ -42,20 +43,65 @@ const SKUpdate& SKNMEA2000Parser::parse(const SKSourceInput& input, const tN2kMs
   if (_sku) {
     delete(_sku);
   }
+  //RONDEBUG
```

Comment:
Did you mean to keep this ;)?

Reply:
I should not make PRs late night....
Sorry for all those careless mistakes.